test/datatype/datatype_test.cpp

```diff
@@ -139,3 +143,12 @@ TEST(Datatype, Size) {
   a.size(4);
   ASSERT_EQ(a.size(), 4ul);
 }
+
+TEST(Datatype, Sign) {
```

Comment:
Again, this test should go with the `Integer` tests, not here. Test it with an actual integer type. Implementation for Enum types is currently in another branch and not yet available here, but once we have it available you will be able to create Enums on the basis of any Integer type. Anyway, should not be needed for this test.